Source/JavaScriptCore/wasm/WasmCallee.cpp

```diff
@@ -81,6 +83,9 @@ inline void Callee::runWithDowncast(const Func& func)
     case CompilationMode::WasmToJSMode:
         func(static_cast<WasmToJSCallee*>(this));
         break;
+    case CompilationMode::OMGForInlineCallMode:
+        RELEASE_ASSERT_NOT_REACHED();
+        break;
```

Comment:
Let's not add this. `CompilationMode` is for Callee. If the compiler is not creating special callee for that, then we should not add it, this is not the data type for that.